src/db2tcl.c

```diff
@@ -88,7 +88,7 @@ Tcl_Interp * interp;
     Tcl_CreateObjCommand (interp, "db2_test", Db2_test,
                        (ClientData) NULL, (Tcl_CmdDeleteProc *) NULL);
 
-    Tcl_PkgProvide (interp, "db2tcl", "2.0.1");
+    Tcl_PkgProvide (interp, "db2tcl", "2.0.1.1");
```

Comment:
Could the two quoted strings be replaced with PACKAGE and PACKAGE_VERSION to avoid repetition?
This is also partially done on line 22.  Curious why the Tcl_PkgProvide() call is effectively duplicated.

Also curious why PACKAGE is defined for Windows only.

Reply:
Good catch, this has been corrected